keps/sig-scheduling/2926-job-mutable-node-affinity/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: Mutable NodeAffinity for Suspended Jobs
+kep-number: 2926
+authors:
+  - "@ahg-g"
+owning-sig: sig-scheduling
```

Comment:
I think it should be `sig-api-machinery` as there is no scheduling code involved.

Reply:
I think the owning sig should probably the one who introduced the NodeAffinity feature, I assume we introduced that validation code and proposed to make it not mutable initially.

Reply:
In any case, an API review is certainly required.

Reply:
SG as long as we have API approval.